composablejs/packages/bootstrap_pallets/src/bootstrap/stakingRewards/index.ts

```diff
@@ -0,0 +1,23 @@
+import { mintAssetsToWallets } from "@composable/bootstrap_pallets/lib";
+import { ApiPromise } from "@polkadot/api";
+import { KeyringPair } from "@polkadot/keyring/types";
+import { createRewardPool } from "@composable/bootstrap_pallets/lib/pallets/stakingRewards";
+import { toStakingRewardPoolConfig } from "@composable/bootstrap_pallets/utils/stakingRewards";
+import BigNumber from "bignumber.js";
+import config from "@composable/bootstrap_pallets/constants/config.json";
+
+export async function bootstrapStakingRewardPools(api: ApiPromise, walletSudo: KeyringPair): Promise<void> {
+    // minting gas
+    // await mintAssetsToWallets(api, [walletSudo], walletSudo, ["1"], toChainUnits("10"));
+
+    for (const pool of config.stakingRewardPools) {
```

Comment:
```suggestion
    // minting gas

    for (const pool of config.stakingRewardPools) {
```
nit: comment can be removed?